lib/propertiesPanel/JSONDataProperty.js

```diff
@@ -96,7 +96,10 @@ const JSONDataProperty = (props) => {
     }
 
     try {
-      JSON.parse(value);
+      const parsed = JSON.parse(value);
+      if (typeof parsed !== 'object') {
```

Comment:
Note:
`typeof JSON.parse('null')`

Reply:
Good catch, I adjusted by using the min-dash `isObject` method. This works as expected now:
![image](https://github.com/camunda/example-data-properties-provider/assets/21984219/dd194848-f490-4d7b-ba73-79ec1fb9879c)